import catchAsyncError from "../../utils/catchAsync";

const createAcademicSemester = catchAsyncError(async (req, res) => {


})


export const academicSemesterControllers = {
    createAcademicSemester,
}